devel/ci/integration/resultsdb/Dockerfile

```diff
@@ -1,22 +1,17 @@
-FROM centos:7
```

Comment:
Is this change related to the waiverdb issue?

Reply:
We don't have fedora-messaging in proper EPEL and we run waiverdb on f29 in prod (https://pagure.io/waiverdb/blob/master/f/Dockerfile#_1).

Reply:
@cverna yes, but this is the resultsdb container, which I would expect to run on EPEL in production (not actually sure, just an expectation).